src/chatlog/chatlog.cpp

```diff
@@ -830,3 +828,13 @@ void ChatLog::focusOutEvent(QFocusEvent* ev)
             lines[i]->selectionFocusChanged(false);
     }
 }
+
+void ChatLog::keyPressEvent(QKeyEvent * event)
+{
+    if(event->matches(QKeySequence::Copy))
+        if(copyAction->isEnabled())
```

Comment:
Actually @zetok I've decided that nested `if`s must always be braced, for if there's any `else` statement, confusion *will* arise.

@kushagrasurana Please fix that here as well.